.mocharc.yml

```diff
@@ -1 +1,10 @@
 recursive: true
+extension:
+  - ts
+spec:
+  - test/**/*.js
+  - test-ts/**/*.spec.ts
```

Comment:
Would it be possible to put everything in `test/`?

Reply:
Yup, consolidated those folders